### ISSUE TITLE
Compatibility with Laravel 6

### ISSUE THREAD

**Issue by trollfalgar** [Kyslik/laravel-filterable#14]
Can you update the package to be compatible with Laravel 6?

**Comment by Kyslik** [comment]
I did not think anyone uses this package; I will see what I can do; definitely will drop Support for L5.* and just go straight for L6.

I'd appreciate a PR if possible though!

**Comment by lachlanhickey** [comment]
This app is literally the back bone of my search functioanlity across many of my applications.. It's certaintly in use!

**Comment by trollfalgar** [comment]
> 
> 
> This app is literally the back bone of my search functioanlity across many of my applications.. It's certaintly in use!

This package is really good.
Thanks!

**Comment by trollfalgar** [comment]
> 
> 
> I did not think anyone uses this package; I will see what I can do; definitely will drop Support for L5.* and just go straight for L6.
> 
> I'd appreciate a PR if possible though!

This package is really good.
Thanks!

**Comment by rtconner** [comment]
Yes please upgrade to laravel 6. Also I love this package a lot, I use it in all my projects.

**Comment by qindj** [comment]
> This app is literally the back bone of my search functioanlity across many of my applications.. It's certaintly in use!

+1

**Comment by phylaxis** [comment]
Hey, I'm having issues with using this branch after upgrading to Laravel 6. I'm getting this error:

```
Symfony \ Component \ HttpKernel \ Exception \ HttpException
Macro '->hasAnyFilter' requires a parameter of a \Kyslik\LaravelFilterable\FilterContract.
```

I'm reviewing all the docs and it seems as if I have everything implemented correctly. Does anyone here have any thoughts on what I might be doing wrong? Thanks in advance for any help.

**Comment by phylaxis** [comment]
Just want to check back in here one more time before I give up and move on. Does anyone know what might be causing this error? I assume this project has been abandoned? If so, what are people using as a replacement / alternative?

**Comment by trollfalgar** [comment]
> Just want to check back in here one more time before I give up and move on. Does anyone know what might be causing this error? I assume this project has been abandoned? If so, what are people using as a replacement / alternative?

The creator abandoned the package.
I recommend search for alternatives.
I refactored my application to use only model scopes, for exemple.
